build/commands/scripts/sync.js

```diff
@@ -19,16 +20,24 @@ program
   .option('--target_android_base <target_android_base>', 'target Android OS level for apk or aab (classic, modern, mono)')
   .option('--init', 'initialize all dependencies')
   .option('--force', 'force reset all projects to origin/ref')
-  .option('--ignore_chromium', 'do not update chromium version even if it is stale')
+  .option('--sync_chromium [arg]', 'force or skip chromium sync (true/false/1/0)', JSON.parse)
+  .option('--ignore_chromium', 'do not update chromium version even if it is stale [deprecated, use --sync_chromium=false]')
+  .option('--sync_chromium_and_delete_unused_deps', 'force chromium sync and delete from the working copy any dependencies that have been removed since the last sync')
```

Comment:
this option seems very verbose, can we just add `-D` like upstream?

Reply:
a resync is required to do a cleanup. previously this option was named `delete_unused_deps`, but it won't work without adding `--sync_chromium`.

we can make it `-D`, but should we do the resync implicitly? maybe it's fine tbh. we can just write this into the help message.